packages/scans/src/action-helper.js

```diff
@@ -14,6 +14,10 @@ function createReadStreamSafe(filename, options) {
     });
 }
 
+function hasOwnProperty(obj, key) {
+    return Object.prototype.hasOwnProperty.call(obj, key);
```

Comment:
Via [`no-prototype-builtins`](https://eslint.org/docs/rules/no-prototype-builtins) via `eslint:recommended`

MDN: [`Object.prototype.hasOwnProperty()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty)

Personally I'd prefer [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining), e.g.:

```diff
diff --git a/packages/scans/src/action-helper.js b/packages/scans/src/action-helper.js
index 49cbc8a..c3a6c8a 100644
--- a/packages/scans/src/action-helper.js
+++ b/packages/scans/src/action-helper.js
@@ -60,7 +60,7 @@ let actionHelper = {

         sites.forEach((site => {
             msg = msg + `${BULLET} Site: [${site["@name"]}](${site["@name"]}) ${NXT_LINE}`;
-            if (hasOwnProperty(site, 'alerts')) {
+            if (site?.alerts) {
                 if (site.alerts.length !== 0) {
                     msg = `${msg} ${TAB} **New Alerts** ${NXT_LINE}`;
                     site.alerts.forEach((alert) => {
```

If we intend on ever doing a `dist/` release with this, `ncc` raised an error:

```
./node_modules/.bin/ncc build
ncc: Version 0.21.1
ncc: Compiling file index.js
Error: Module parse failed: Unexpected token (63:21)
File was processed with these loaders:
 * ./node_modules/@zeit/ncc/dist/ncc/loaders/empty-loader.js
 * ./node_modules/@zeit/ncc/dist/ncc/loaders/relocate-loader.js
 * ./node_modules/@zeit/ncc/dist/ncc/loaders/shebang-loader.js
You may need an additional loader to handle the result of these loaders.
|         sites.forEach((site => {
|             msg = msg + `${BULLET} Site: [${site["@name"]}](${site["@name"]}) ${NXT_LINE}`;
>             if (site?.alerts) {
|                 if (site.alerts.length !== 0) {
|                     msg = `${msg} ${TAB} **New Alerts** ${NXT_LINE}`;
    at evalmachine.<anonymous>:1:1416387
    at eval (eval at create (evalmachine.<anonymous>:1:274053), <anonymous>:13:1)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
```

If we update `ncc` from 0.21.0 to 0.33.0 it fixes it:

`npm uninstall @zeit/ncc && npm install --save-dev @vercel/ncc`

Reply:
See #27